src/app/app.component.ts

```diff
@@ -7,6 +7,8 @@ import { ApplicationInsights } from '@microsoft/applicationinsights-web';
 import { AngularPlugin } from '@microsoft/applicationinsights-angularplugin-js';
 import { environment } from 'src/environments/environment';
 import { LocalStorageKeys } from '@app/shared/models';
+import { EnvironmentType } from '../environments/model';
+import * as jQuery from 'jquery';
```

Comment:
You can't import like that or jQuery will be loaded everywhere. If anything, you need a dynamic import directly inside of the `if` statement.